NDTensors/src/backup/arraystorage/diagonalarray/storage/contract.jl

```diff
@@ -2,16 +2,20 @@ using .SparseArrayInterface: densearray
 using .DiagonalArrays: DiagIndex, diaglength
 
 # TODO: Move to a different file.
-Unwrap.parenttype(::Type{<:DiagonalArray{<:Any,<:Any,P}}) where {P} = P
+function TypeParameterAccessors.position(
+  ::Type{<:DiagonalArray}, ::typeof(TypeParameterAccessors.parenttype)
```

Comment:
Same comment, let's load `using TypeParameterAccessors: parenttype` since this is getting quite long.

Reply:
Makes sense, I added this to the top and the comment above in `Blocksparse/unwrap`, thanks!